server/py/services/api/launcher.py

```diff
@@ -697,20 +697,31 @@ def _validate_retry(runtime_kind: str, retry: Optional["mlrun.model.Retry"]):
                 )
 
     # TODO In ML-11600, implement token name resolution and validation + tests
-    def _enrich_and_validate_auth_token_name(self, run: mlrun.run.RunObject):
-        auth = run.spec.auth or {}
+    def enrich_and_validate_auth_token_name_on_object(
```

Comment:
```suggestion
    def enrich_and_validate_auth_token_name(
```
why renaming? also, function name should reflect what it does (enriches/validates the auth token) the added information is subjected to the argument - which can be omitted from the name for redundancy or (TMI)